fe/fe-core/src/main/java/com/starrocks/StarRocksFE.java

```diff
@@ -277,26 +277,34 @@ private static void checkCommandLineOptions(CommandLineOptions cmdLineOpts) {
         // go on
     }
 
-    private static boolean createAndLockPidFile(String pidFilePath) throws IOException {
+    private static boolean createAndLockPidFile(String pidFilePath) {
         File pid = new File(pidFilePath);
-        try (RandomAccessFile file = new RandomAccessFile(pid, "rws")) {
-            FileLock lock = file.getChannel().tryLock();
-            if (lock == null) {
-                return false;
-            }
+        for (int i = 0; i < 3; i++) {
+            try (RandomAccessFile file = new RandomAccessFile(pid, "rws")) {
```

Comment:
It's better to find the core problem with the lock held.
Why the exit of FE can not release the lock?

Reply:
This problem only occurred in our daily env, but I have not found the root cause. This PR is try to fix this bug, and print more error log if get lock failed.

Reply:
If it's a temporary solution, you can add your PR into a daily environment instead of the main branch.
After finding the root cause, you can make a pull request to address the problem ultimately.

Reply:
But daily can not run this pr every day. It is not always reproducible.